code/modules/clothing/glasses/glasses.dm

```diff
@@ -279,6 +279,16 @@
 	icon_state = "redglasses"
 	item_state = "redglasses"
 
+/obj/item/clothing/glasses/godeye
+	name = "eye of god"
+	desc = "A strange eye, said to have been torn from an omniscient creature that used to roam the wastes."
+	icon_state = "godeye"
+	item_state = "godeye"
+	vision_flags = SEE_TURFS|SEE_MOBS|SEE_OBJS
+	darkness_view = 8
+	scan_reagents = 1
+	flags = 2

```

Comment:
Ah, yes, I'd like for it to be stuck to the person's face. I've always liked the theme of cursed items on lavaland.